srcJava/com/rti/perftest/harness/PerfTest.java

```diff
@@ -756,6 +754,118 @@ else if ("-cft".toLowerCase().startsWith(argv[i].toLowerCase()))
         return true;
     }
 
+    private void printConfiguration() {
+
+        StringBuilder sb = new StringBuilder();
+
+        sb.append("\nPerftest Configuration:\n");
+
+        // Throughput/Latency mode
+        if (_isPub) {
+            sb.append("\tMode: ");
+            if (_latencyTest) {
+                sb.append("Latency (Ping-Pong test)\n");
+            } else {
+                sb.append("Throughput (Use \"-latencyTest\" for Latency Mode)\n");
+            }
+
+            sb.append("\tLatency count: 1 latency sample every ");
+            sb.append(_latencyCount);
+            sb.append("\n");
+        }
+
+        // Reliable/Best Effort
+        sb.append("\tReliability: ");
+        if (_isReliable) {
+            sb.append("Reliable\n");
+        } else {
+            sb.append("Best Effort\n");
+        }
+
+        // Keyed/Unkeyed
+        sb.append("\tKeyed: ");
+        if (_isKeyed) {
+            sb.append("Yes\n");
+        } else {
+            sb.append("No\n");
+        }
+
+        // Publisher/Subscriber and Entity ID
+        if (_isPub) {
+            sb.append("\tPublisher ID: ");
+            sb.append(pubID);
+            sb.append("\n");
+        } else {
+            sb.append("\tSubscriber ID: ");
+            sb.append(subID);
+            sb.append("\n");
+        }
+
+        // Scan/Data Sizes
+        sb.append("\tData Size: ");
+        if (_isScan) {
+            for (int i = 0; i < _scanDataLenSizes.size(); i++ ) {
+                sb.append(_scanDataLenSizes.get(i));
+                if (i == _scanDataLenSizes.size() - 1) {
+                    sb.append("\n");
+                } else {
+                    sb.append(", ");
+                }
+            }
+        } else {
+            sb.append(_dataLen);
+            sb.append("\n");
+        }
+
+        // Batching
+        sb.append("\tBatching: ");
+        if (_batchSize != 0) {
+            sb.append(_batchSize);
+            sb.append(" Bytes (Use \"-batchSize 0\" to dissable batching)\n");
+        } else {
+            sb.append("No (Use \"-batchSize\" to setup batching)\n");
+        }
+
+        // Listener/WaitSets
+        sb.append("\tReceive using: ");
+        if (_useReadThread) {
+            sb.append("WaitSets\n");
+        } else {
+            sb.append("Listeners\n");
+        }
+
+        if (_isPub) {
```

Comment:
why not write the comment before the condition as in the previous condition

Reply:
Got it!